css/feed.css

```diff
@@ -1,4 +1,6 @@
 .org-organization-admin-pages-entrypoint-card__card,
-.feed-identity-module__widgets {
+.feed-identity-module__widgets,
+.artdeco-toast-item--visible,
```

Comment:
This is the notification on the bottom-left of the page that stays there until you click the x to close it.